client/state/analytics/middleware.js

```diff
@@ -15,13 +21,22 @@ const eventServices = {
 
 const pageViewServices = {
 	ga: ( { url, title } ) => analytics.ga.recordPageView( url, title ),
-	default: ( { url, title } ) => analytics.pageView.record( url, title )
+	'default': ( { url, title } ) => analytics.pageView.record( url, title ),
+};
+
+const loadTrackingTool = ( trackingTool, state ) => {
+	const trackUser = ! navigator.doNotTrack;
```

Comment:
can we earmark this and come back to it later. I think I was talking with @markryall about this earlier and at some point we should make a simple conditional include of the analytics middleware only if `doNotTrack` isn't true. we still do have an issue with people calling this code directly but hopefully over time we can eliminate all those direct calls and keep it that way.